### PR TITLE
GEN-1520 | Prevent site crawling for all widget pages

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Prevent search engine indexing of all _dynamic_ widget pages

- Place all dynamic widget pages under "widget/run" namespace so they are easy to target in robot.txt

## Justify why they are needed

- Dito

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3667** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3667?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3668** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3668?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3669** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3669?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3670** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3670?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3671** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3671?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3672** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3672?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #3673** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3673?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3673?utm_source=stack-comment).

**Review by gustaveen — APPROVED** [review]

File: apps/store/public/robots.txt [hunk 1]
```diff
@@ -8,5 +8,7 @@ Disallow: /se-en/confirmation
 Disallow: /country-selector
 Disallow: /reusable-blocks
 Disallow: /referrals
+Disallow: /se/widget/run
```
Comment (gustaveen) [inline apps/store/public/robots.txt:11]: Just wondering, is `run` a acronym for something? :)

**Review by gustaveen — APPROVED** [review]
Looks good! 🙌 Just had a question about the path :)

File: apps/store/public/robots.txt (hunk 1 above)
Comment (robinandeer) [reply apps/store/public/robots.txt:11]: haha no - I couldn't use "flows" so had to come up with something else

"run" is just representing an instance of the widget

but I'm open to suggestions 😬

**Comment by robinandeer** [comment]
### Merge activity

* **Dec 1, 1:06 PM**: @robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3673).

**Pull request merged**